src/js/models/maps/GeoPoint.js

```diff
@@ -118,19 +124,56 @@ define(["backbone", "models/maps/GeoUtilities"], function (
        */
       validate: function (attrs) {
         if (attrs.latitude < -90 || attrs.latitude > 90) {
-          return "Invalid latitude. Must be between -90 and 90.";
+          return { latitude: "Invalid latitude. Must be between -90 and 90." };
         }
 
         if (attrs.longitude < -180 || attrs.longitude > 180) {
-          return "Invalid longitude. Must be between -180 and 180.";
+          return {
+            longitude: "Invalid longitude. Must be between -180 and 180."
+          };
         }
 
         // Assuming height is in meters and can theoretically be below sea
         // level. Adjust the height constraints as needed for your specific
         // application.
         if (typeof attrs.height !== "number") {
-          return "Invalid height. Must be a number.";
+          return { height: "Invalid height. Must be a number." };
+        }
+      },
+    },
+    {
```

Comment:
I had a go at using `class` instead of a `Backbone.X.extend` but some things start to fall apart due to the ways Backbone handles inheritance ([an article I shared with Yvonne](https://benmccormick.org/2015/04/06/221944.html)), so this was a very handy alternative.